spec/features/admin/reports/enterprise_fee_summaries_spec.rb

```diff
@@ -96,60 +98,86 @@
     end
   end
 
-  describe "smoke test for generation of report based on permissions" do
-    before do
-      visit spree.new_admin_reports_enterprise_fee_summary_path
+  describe "csv downloads" do
+    around do |example|
+      clear_downloads
+      example.run
+      clear_downloads
     end
```

Comment:
How about adding a method similar to `performing_deliveries` in `EmailHelper` for this?

Reply:
ah, nice. I'll do that.

Reply:
done